code/game/gamemodes/cult/runes.dm

```diff
@@ -514,13 +528,19 @@
 	corpse_to_raise.key = ghost.key	//the corpse will keep its old mind! but a new player takes ownership of it (they are essentially possessed)
 									//This means, should that player leave the body, the original may re-enter
 	usr.say("Pasnar val'keriam usinar. Savrae ines amutan. Yam'toth remium il'tarat!")
-	corpse_to_raise.visible_message("<span class='warning'>[corpse_to_raise]'s eyes glow with a faint red as he stands up, slowly starting to breathe again.</span>", \
-	"<span class='warning'>Life... I am alive again...</span>", \
-	"<span class='warning'>You hear a faint, slightly familiar whisper.</span>")
-	body_to_sacrifice.visible_message("<span class='warning'>[body_to_sacrifice] is torn apart, a black smoke swiftly dissipating from his remains!</span>", \
-	"<span class='warning'>You feel as your blood boils, tearing you apart.</span>", \
-	"<span class='warning'>You hear a thousand voices, all crying in pain.</span>")
-	body_to_sacrifice.gib()
+	if (body_to_sacrifice)
+		corpse_to_raise.visible_message("<span class='warning'>[corpse_to_raise]'s eyes glow with a faint red as he stands up, slowly starting to breathe again.</span>", \
+		"<span class='warning'>Life... I am alive again...</span>", \
+		"<span class='warning'>You hear a faint, slightly familiar whisper.</span>")
+		body_to_sacrifice.visible_message("<span class='warning'>[body_to_sacrifice] is torn apart, a black smoke swiftly dissipating from his remains!</span>", \
+		"<span class='warning'>You feel as your blood boils, tearing you apart.</span>", \
+		"<span class='warning'>You hear a thousand voices, all crying in pain.</span>")
```

Comment:
Isn't there an ominous class? Seems like if it's something spooky like this it'd be more deserving of ominous or spooky, rather than the usual attack-text red